openmeetings-util/src/main/java/org/apache/openmeetings/util/OmFileHelper.java

```diff
@@ -101,6 +101,7 @@ private OmFileHelper() {}
 	public static void setOmHome(File home) {
 		OM_HOME = home;
 		final String dataDir = System.getProperty("DATA_DIR");
+		log.info("Using file locations - OM_HOME: {} DATA_DIR: {}", OM_HOME, dataDir);
```

Comment:
I would move it _after_ else and change as

`log.info("Using file locations - OM_HOME: {} DATA_DIR: {}", OM_HOME, DATA_DIR);`

IMO it would be better :)

Reply:
funny how I coded the same line without reading this.